src/services/iasql.ts

```diff
@@ -86,14 +83,11 @@ export async function connect(
     dbUser = dbGen[0];
     const dbPass = dbGen[1];
     dbId = dbMan.genDbId(dbAlias);
-    const hasCredentials = !!awsAccessKeyId && !!awsSecretAccessKey;
     const metaDb = new IasqlDatabase();
     metaDb.alias = dbAlias;
     metaDb.pgUser = dbUser;
-    metaDb.region = awsRegion;
     metaDb.pgName = dbId;
     metaDb.directConnect = directConnect
-    metaDb.isReady = hasCredentials;
```

Comment:
So removing this here will make the field in the database always default to true, which would mean that databases that use direct connect will be immediately `ready` in the dashboard when they are not.

Reply:
So, kinda?

This sort of metadata that can be derived from the actual database should probably be there instead of here. Two source of truth, y'know. Someone could delete the `aws_account` record and our UI would still show it as ready when it isn't.

Also saying kinda because if/when we have multiple module "trees" the "is ready" condition becomes more complex, and we very likely want to move it out of here entirely and push it to the dashboard (since the database is technically usable on connect).

Reply:
Took care of it as suggested in the paired dashboard change